gradle.properties

```diff
@@ -1,2 +1,3 @@
 org.gradle.jvmargs=-XX:MaxPermSize=512m
 org.gradle.caching=true
+android.enableD8=true
```

Comment:
No need to add this.

Reply:
I have removed d8 from the library modules 👍